tests/simulate/test_hoomd.py

```diff
@@ -2,27 +2,19 @@
 import tempfile
 import unittest
 
-from parameterized import parameterized_class
-
-try:
-    import gsd.hoomd
-
-    _found_gsd = True
-except ImportError:
-    _found_gsd = False
+import gsd.hoomd
```

Comment:
Won't this trigger an import error if not installed? Do we want the test to skip if it doesn't have hoomd or freud but raise an error without gsd.

Reply:
Yes, we would want an error to be raised so the developer realizes they should have installed the testing dependencies.